plugins/node/instrumentation-runtime-node/src/instrumentation.ts

```diff
@@ -13,83 +13,75 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { EventLoopUtilization, performance } from 'node:perf_hooks';
-const { eventLoopUtilization } = performance;
-
 import { InstrumentationBase } from '@opentelemetry/instrumentation';
 
-/** @knipignore */
-import { PACKAGE_NAME, PACKAGE_VERSION } from './version';
 import { RuntimeNodeInstrumentationConfig } from './types';
+import { MetricCollector } from './types/metricCollector';
+import { EventLoopUtilizationCollector } from './metrics/eventLoopUtilizationCollector';
+import { EventLoopDelayCollector } from './metrics/eventLoopDelayCollector';
+import { GCCollector } from './metrics/gcCollector';
+import { HeapSpacesSizeAndUsedCollector } from './metrics/heapSpacesSizeAndUsedCollector';
+import { ConventionalNamePrefix } from './types/ConventionalNamePrefix';
+import { EventLoopTimeCollector } from './metrics/eventLoopTimeCollector';
+import { PACKAGE_VERSION, PACKAGE_NAME } from './version';
```

Comment:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2477 added a new tool named `knip` to lint dependencies and it fails to find the `version.ts` file. To avoid this error in CI you should add the following comment above this line

```js

/** @knipignore */
```